docs/guides/js-waku/lightpush-filter.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Send and Receive Messages Using Light Push and Filter
+---
+
+This guide provides detailed steps to create a light node, send messages using the [Light Push protocol](/overview/concepts/protocols#light-push), and receive messages using the [Filter protocol](/overview/concepts/protocols#filter).
+
+## Create a Waku Node
+
+Set up a Waku node by creating a light node, connecting to network peers with `Light Push` and `Filter` enabled, choosing a [content topic](/overview/concepts/content-topics), and creating an `encoder` and `decoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+
+```js
+import {
+    createLightNode,
+    waitForRemotePeer,
+    Protocols,
+    createEncoder,
+    createDecoder
+} from "@waku/sdk";
+
+// Create and start a light node
+const node = await createLightNode({ defaultBootstrap: true });
+await node.start();
+
+// Wait for a successful peer connection
+await waitForRemotePeer(node, [
+	Protocols.LightPush,
+	Protocols.Filter,
+]);
+
+// Choose a content topic
+const contentTopic = "/light-guide/1/message/proto";
+
+// Create a message encoder and decoder
+const encoder = createEncoder(contentTopic);
+const decoder = createDecoder(contentTopic);
+```
+
+## Create a Message Structure
+
+Create your application's message structure using [Protobuf's valid message](https://github.com/protobufjs/protobuf.js#usage) fields:
+
+```js
+import protobuf from "protobufjs";
+
+// Create a message structure using Protobuf
+const ChatMessage = new protobuf.Type("ChatMessage")
+	.add(new protobuf.Field("timestamp", 1, "uint64"))
+	.add(new protobuf.Field("sender", 2, "string"))
+	.add(new protobuf.Field("message", 3, "string"));
+```
+
+:::info
```

Comment:
Is it some md marking? 
For me it's getting rendered like this:
![image](https://github.com/waku-org/docs.waku.org/assets/118575614/1b25a598-acb4-4ffa-8e1d-a8db256d0296)

Reply:
Same for `:::tip` down in this file.

Reply:
Its called an [admonition](https://docusaurus.io/docs/markdown-features/admonitions). They provide custom styling for tip/info boxes etc. You can view the full preview here: https://waku-guide-git-js-waku-docs-logosdao.vercel.app/guides/js-waku/

<img width="879" alt="Screenshot 2023-07-10 at 10 27 52" src="https://github.com/waku-org/docs.waku.org/assets/47832826/2e2a9c5b-cfbd-49b6-a916-2f854c829f57">

<img width="855" alt="Screenshot 2023-07-10 at 10 28 07" src="https://github.com/waku-org/docs.waku.org/assets/47832826/c0d77f6d-1c18-446f-b5fe-1fda92eee9ae">